PowerPointLabs/PowerPointLabs/AudioMisc/Audio.cs

```diff
@@ -74,6 +99,9 @@ public void EmbedOnSlide(PowerPointSlide slide, int clickNumber)
                     slide.DeleteShapesWithPrefixTimelineInvariant(Name);
 
                     audioShape.Name = shapeName;
+                    
+                    // tag item with type
+                    audioShape.Tags.Add(AudioTypeTagName, Type.ToString());

```

Comment:
The type is already defined in the constructor when it's checked from the shape, line 52 of Audio.cs.

And I'm avoiding the use of sampling rate as part of the issue I'm solving.